sdk/src/main/java/com/horizen/account/api/rpc/types/EthereumBlockView.java

```diff
@@ -58,6 +58,7 @@ public EthereumBlockView(Long blockNumber, String hash, boolean hydratedTx, Acco
         this.gasUsed = Numeric.toHexStringWithPrefix(BigInteger.valueOf(blockHeader.gasUsed()));
         this.timestamp = Numeric.prependHexPrefix(Long.toHexString(block.timestamp()));
         this.baseFeePerGas = Numeric.toHexStringWithPrefix(blockHeader.baseFee());
+        this.difficulty = Numeric.encodeQuantity(BigInteger.valueOf(blockNumber));
```

Comment:
@davehorizen hey, look at this lines https://github.com/HorizenOfficial/Sidechains-SDK/pull/609/files#diff-361cfee08bad4e5f175ad4eb76ba8face23db9e0283ac73b1e4718f0e371b0f5R37-R43
After some iterations, we found the proper values we need to return via RPC.